docs/plugins.rst

```diff
@@ -0,0 +1,131 @@
+
+These data are acquired from the National Water Quality Monitoring Council `Water Quality Portal <https://www.waterqualitydata.us/>`_.
+specifying the provider STORET.
+
+**Data Usage** See citation information at `WQP User Guide <https://www.waterqualitydata.us/portal_userguide/>`_. Requests to the WQP web services are logged. See :doc:`/quick_guide`.
+
+Section 1: EPA Data Source Configuration
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+None.
+Data are publicly available and accessed via the `Water Quality Portal Web Services <https://www.waterqualitydata.us/webservices_documentation/>`_.
+Please follow data usage guidelines at `User Guide <https://www.waterqualitydata.us/portal_userguide/>`_ .
+
+
+Section 2: Using the EPA plugin in BASIN-3D
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+1. Install basin3d using the instructions in the :doc:`/getting_started`.
+
+2. Import the EPA WQX plugin:
+
+.. code-block::
+
+    >>> from basin3d.plugins import epa
+
+3. Register the plugin (and any other you have imported):
+
+.. code-block::
+
+    >>> synthesizer = synthesis.register()
+
+4. Find the available monitoring feature IDs (aka location identifiers). Note: These monitoring feature IDs are required parameters for BASIN-3D data requests
+
+Option 1: Specify a `USGS HUC <https://water.usgs.gov/GIS/huc.html>`_ 2, 4, 6, or 8-digit code in the parent_feature argument. Prefix the HUC identifier with "EPA-".
+The BASIN-3D USGS plugin can be used to find USGS HUC information and identifiers:
+
+.. code-block::
+
+    >>> monitoring_features = synthesizer.monitoring_features(parent_feature=['EPA-14020001'])
+    >>> for monitoring_feature in monitoring_features:
+    >>>     print(f'{monitoring_feature.id} --- {monitoring_feature.name}')
+    EPA-21COL001-000058 --- TAYLOR RIVER AT ALMONT
+    EPA-21COL001-000078 --- EAST RIVER AT CONFL. WITH TAYLOR
+    EPA-21COL001-000150 --- SLATE RIVER ABOVE COAL CREEK
+    ...
+
+Option 2: Specify one or more EPA Site IDs, prefixed by "EPA-":
+
+.. code-block::
+
+    >>> monitoring_features = synthesizer.monitoring_features(parent_feature=['EPA-CORIVWCH_WQX-176', 'EPA-11NPSWRD_WQX-BLCA_09128000'])
+    >>> for monitoring_feature in monitoring_features:
+    >>>     print(f'{monitoring_feature.id} --- {monitoring_feature.name}')
+    EPA-11NPSWRD_WQX-BLCA_09128000 --- GUNNISON RIVER BELOW GUNNISON TUNNEL, CO
+    EPA-CORIVWCH_WQX-176 --- Uncompahgre -  Confluence Park
+
+
+5. Request time series data. Query argument aggregation_duration supports "NONE" or "DAY". See `epa_mapping.py <https://github.com/BASIN-3D/basin3d/blob/main/basin3d/plugins/epa_mapping.py>`_:
+
+.. code-block::
+
+    >>> measurement_timeseries_tvp_observations = synthesizer.measurement_timeseries_tvp_observations(monitoring_feature=['EPA-CCWC-COAL-26', 'EPA-CCWC-MM-29 WASH #3'], observed_property=['As', 'WT', 'DO'], start_date='2010-01-01', end_date='2011-01-01', aggregation_duration='NONE')
```

Comment:
I suggest giving an example of how to work with the return value.  We should note in a new ticket that it is very hard to use the API reference to figure out what the properties of a measurement tvp time series is. It took me a bit to figure out what I should do with the return value.  

If we give this example, should we link to the section of the documentation that discusses the "Could not parse" message.


```suggestion
>>> measurement_timeseries_tvp_observations = synthesizer.measurement_timeseries_tvp_observations(monitoring_feature=['EPA-CCWC-COAL-26', 'EPA-CCWC-MM-29 WASH #3'], observed_property=['As', 'WT', 'DO'], start_date='2010-01-01', end_date='2011-01-01', aggregation_duration='NONE')
>>> for m in measurement_timeseries_tvp_observations:
...     for r in m.result.value:
...         print(f"feature_of_interest:'{m.feature_of_interest}' observed_property:{m.observed_property} timestamp:{r.timestamp} value:{r.value} {m.unit_of_measurement}" )
... 
Could not parse expected numerical measurement value <0.500
Could not parse expected numerical measurement value <2.50
Could not parse expected numerical measurement value <0.500
Could not parse expected numerical measurement value <0.500
feature_of_interest:'EPA-CCWC-COAL-26' observed_property:As timestamp:2010-08-17T11:30:00-07:00 value:14.4 ug/L
feature_of_interest:'EPA-CCWC-COAL-26' observed_property:As timestamp:2010-10-12T14:27:00-07:00 value:14.0 ug/L
feature_of_interest:'EPA-CCWC-COAL-26' observed_property:As timestamp:2010-10-12T14:27:00-07:00 value:9.44 ug/L
feature_of_interest:'EPA-CCWC-COAL-26' observed_property:As timestamp:2010-05-18T16:21:00-07:00 value:3.38 ug/L
feature_of_interest:'EPA-CCWC-COAL-26' observed_property:As timestamp:2010-10-12T14:27:00-07:00 value:9.73 ug/L
feature_of_interest:'EPA-CCWC-COAL-26' observed_property:As timestamp:2010-04-20T13:43:00-07:00 value:4.19 ug/L
feature_of_interest:'EPA-CCWC-COAL-26' observed_property:As timestamp:2010-10-12T14:27:00-07:00 value:11.7 ug/L
feature_of_interest:'EPA-CCWC-COAL-26' observed_property:As timestamp:2010-08-17T11:30:00-07:00 value:14.3 ug/L
feature_of_interest:'EPA-CCWC-COAL-26' observed_property:As timestamp:2010-05-18T16:21:00-07:00 value:3.98 ug/L
feature_of_interest:'EPA-CCWC-COAL-26' observed_property:As timestamp:2010-08-17T11:30:00-07:00 value:15.1 ug/L
feature_of_interest:'EPA-CCWC-COAL-26' observed_property:WT timestamp:2010-04-20T13:43:00-07:00 value:0.8 deg C
feature_of_interest:'EPA-CCWC-MM-29 WASH #3' observed_property:WT timestamp:2010-04-20T15:00:00-07:00 value:1.52 deg C
feature_of_interest:'EPA-CCWC-MM-29 WASH #3' observed_property:DO timestamp:2010-04-20T15:00:00-07:00 value:14.28 mg/L
feature_of_interest:'EPA-CCWC-COAL-26' observed_property:DO timestamp:2010-04-20T13:43:00-07:00 value:10.78 mg/L
Could not parse expected numerical measurement value <0.500
Could not parse expected numerical measurement value <2.50
Could not parse expected numerical measurement value <0.500
Could not parse expected numerical measurement value <0.500

```

Reply:
done. And added text to reference the could not parse msg.